.travis.yml

```diff
@@ -1,12 +1,17 @@
 # WARNING:
 # Do not use this file as a template for you repository TravisCI configuration
 # Instead use the files under the "/sample_files" directory for that purpose.
-sudo: required
 language: python
 
 python:
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - expect-dev
```

Comment:
I could be wrong but probably because of the colour log? Odoo doesn't have a flag to force coloured log and tee will make it think it's not in a tty, therefore it will not use colours, which makes debugging a failing test a pain.